src/views/v2/tickets/item/content/title.php

```diff
@@ -36,7 +37,7 @@
 <div <?php tribe_classes( $title_classes ); ?> >
 	<?php if ( $is_mini ) : ?>
 		<div <?php tribe_classes( $event_title_classes ); ?> >
-			<?php echo esc_html( get_the_title( $post_id ) ); ?>
+			<?php the_title(); ?>
```

Comment:
We want to be able to pass the $post_id for the specific post to get the title. That was an intentional thing, as this might not be strictly shown on the same page the tickets shortcode/block itself appears.

Reply:
it's currently set to `echo get_the_title( $post_id );` so let me know if that needs to be changed